Fix PHP5.6 compatibility.

RoySegall opened the pull request:
#7931

rbran100 commented on openscholar/modules/harvard/harvard_courses/harvard_courses.context.inc:
```diff
@@ -1,55 +0,0 @@
-/**
- * Implements hook_context_default_contexts().
- */
-function harvard_courses_context_default_contexts() {
```
Why did you remove this?

rbran100 commented:
What is foo.php?

RoySegall commented:
Hmm... I removed the wrong file and them removed foo.php again. I had questioned my self why foo.php is still there though I removed it.

Any way it was a file that was there for an unkown reason.
